src/apigateway/router.js

```diff
@@ -64,8 +64,15 @@ class Router {
             await endpoint[method](request, response);
         }
 
-        if (!response.hasErrors && endpoint.requirements && endpoint.requirements[method]) {
-            await responseValidator.isValid(endpoint.requirements[method]);
+        const responseBodyValidationRuleKey = 'responseBody';
+        const hasResponseValidationRule =
+            endpoint.requirements &&
+            endpoint.requirements[method] &&
+            endpoint.requirements[method][responseBodyValidationRuleKey];
+
+        if (!response.hasErrors && hasResponseValidationRule) {
+            const rule = endpoint.requirements[method][responseBodyValidationRuleKey];
```

Comment:
This seems a little different the above, for consistency sakes we should just keep it like the above:

```suggestion
        if (!response.hasErrors && endpoint.requirements && endpoint.requirements[method] && endpoint.requirements[method]['responseBody']) {
            const rule = endpoint.requirements[method]['responseBody'];
```

If this is too long, just update the `package.json` to use `"ecmaVersion": 2020` in the `eslintConfig` object. Then you can use optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining